snippets/assert-attribute-must-be-set.js

```diff
@@ -0,0 +1,48 @@
+/* Usage of this snippet:
+ *   In the case when you want to confirm an element having a specified attribute.
+ *
+ *   Change the values written below.
+ *    selector: A string of a selector to pinpoint the element.
+ *    attrName: A string of the attribute which you want to assert its value.
+ *    errMessage:
+ *      A string of the error message which you want to show when it fails.
+ *      It will be a default string if you don't change from <TODO: REPLACE>.
+ *
+ * このスニペットの使い方:
```

Comment:
ユニペットを利用しているのは自明なのでちょっと冗長に感じました。
```suggestion
 * 使い方:
```